### ISSUE TITLE
Add alt text to images on all workshops

### ISSUE THREAD

**Issue by lachlanjc** [hackclub/hackclub#1263]
Hack Club’s curriculum should be accessible, no exceptions. The new workshops are missing alt text for screenreader users on all images:

- [ ] Colorful Grammar
- [ ] Rickroll
  - [ ] Rickroll Leader
- [ ] Sound Galaxy
- [ ] Splatter Paint
- [ ] Teachable Machine

From the archive, an incomplete list (search `![](` to find them all) of workshops with uncaptioned images included:

- [ ] Collaborative sketch
- [ ] Dashboard
- [ ] Dodge
- [ ] Find Bigfoot
- [ ] Personal website
- [ ] Platformer
- [ ] Wikibot

**Comment by prsmahajan** [comment]
Hi, i would like to work on this issue, if this is open can you suggest me ways to get started?
